app/src/main/java/com/rockthevote/grommet/data/db/model/Session.kt

```diff
@@ -12,7 +12,7 @@ import java.util.*
  */
 @Entity(tableName = "session")
 data class Session(
-        @PrimaryKey(autoGenerate = true)
+        @PrimaryKey
```

Comment:
If we only ever expect one instance of `Session` in the database, we don't need to auto generate the ID, we can statically set it.

Reply:
feels gross but you're right and will probably help guard against bugs